cfn-resources/datalakes/cmd/resource/resource.go

```diff
@@ -272,98 +260,94 @@ func flattenDataLakeProcessRegion(processRegion *mongodbatlas.DataProcessRegion)
 	return nil
 }
 
-func flattenDataLakeStorageDatabases(databases []mongodbatlas.DataLakeDatabase) []DataLakeDatabaseView {
+func flattenDataLakeStorageDatabases(databases []admin.DataLakeDatabaseInstance) []DataLakeDatabaseView {
 	database := make([]DataLakeDatabaseView, len(databases))
 	for ind := range databases {
 		database = append(database, DataLakeDatabaseView{
-			Name:                   &databases[ind].Name,
+			Name:                   databases[ind].Name,
 			Collections:            flattenDataLakeStorageDatabaseCollections(databases[ind].Collections),
 			Views:                  flattenDataLakeStorageDatabaseViews(databases[ind].Views),
-			MaxWildcardCollections: castNO64(databases[ind].MaxWildcardCollections),
```

Comment:
this looks like aws.IntValue or similar ones

Reply:
here I removed `castNO64` as it was not longer needed. Do I need to do anything as part of this comment?